examples/default/example.tf

```diff
@@ -1,5 +1,36 @@
 provider "aws" {
-  region = "eu-west-1"
+  version = "1.20.0"
```

Comment:
Maybe bump to `1.30.0` which is the latest version?

Reply:
1.30.0 (August 02, 2018) Will update